docs/sphinx/development/continuous_integration.rst

```diff
@@ -90,3 +92,140 @@ Explanation:
 * ``-u "${UID}"`` changes the UID inside the container from root to the current user
 * ``-v "${PWD}:${PWD}"`` mounts the current directory into the container at the same path
 * ``-w "${PWD}"`` chdirs into that directory
+
+Android environment
+===================
+
+The ``env-android`` images provide a full android environment.
+In particular the following packages are availables:
+
+* android platforms
+* android build tools
+* android ndk
+* android cmake
+* android emulator
+* adb and platform tools
+
+For reducing image size and have more granularity over various tools versions, those packages are installed only when container runs, i.e. at container entrypoint.
+
+The following environment variables can be passed at container run for choosing a specified version:
+
+* API
+* BUILD_TOOLS_VERSION
+* NDK_VERSION
+* CMAKE_VERSION
+
+Example:
+
+.. code::
+
+    $ docker run -t --rm -v "${PWD}:${PWD}" -w "${PWD}" --env API=28 \
```

Comment:
Since people will copy-paste this example, maybe also add `-v android-sdk:/sdk` to it?

Reply:
Yes sure